mne_bids/utils.py

```diff
@@ -55,38 +55,40 @@ def get_kinds(bids_root):
     return kinds
 
 
-def get_values_for_key(bids_root, key):
-    """Get list of values for a key in a BIDS dataset.
+def get_entity_vals(bids_root, entity_key):
+    """Get list of values associated with an `entity_key` in a BIDS dataset.
 
     BIDS file names are organized by key-value pairs called "entities" [1]_.
+    With this function, you can get all values for an entity indexed by its
+    key.
 
     Parameters
     ----------
     bids_root : str
         Path to the root of the BIDS directory.
-    key : str
-        The name of the key to search for. Can be one of
+    entity_key : str
+        The name of the entity key to search for. Can be one of
```

Comment:
Can you add a tiny doc example below to show what it will return?

Reply:
done in https://github.com/mne-tools/mne-bids/pull/260/commits/0a4ba164a25163791e8612a7191f1a8688b7f5ae